.github/workflows/ci.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         # ocaml compiler 4.14.1 does not have an option to build in 32 bit mode anymore.
         ocaml_version: [4.14.1, 5.3.0]
     container:
-      image: ocamlcross/windows-x64-base:${{ matrix.ocaml_version }}
```

Comment:
I think it's nice to keep indicating the arch here. Helps the user preemptively making choices.

Reply:
agreed, dropped